doc/changes/DM-33932.removal.md

```diff
@@ -0,0 +1,2 @@
+Remove the ``spectraction`` dataset type. This was a temporary dataset type added for convenience during development, which was a roll-up-and-pickle of all the potentially relevant parts of the extraction. All the necessary information is now stored inside the ``SpectractorSpectrum`` dataset.
```

Comment:
You are removing the `Spectraction` storage class here. Butler does not know about any `spectraction` dataset type so you should tweak this removal message.

Reply:
Done, thanks.